src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/help/contents/ascanalpha.html

```diff
@@ -9,6 +9,10 @@
 <H1>Active Scan Rules - alpha</H1>
 The following alpha quality active scan rules are included in this add-on:
 
+<H2>Apache Range Header DoS (CVE-2011-3192)</H2>
+Tests to see if the server is vulnerable to the Apache Range Header Denial of Service issue. By requesting 11 different byte ranges, one more than should be
```

Comment:
Yeah, I read that as well, the "more than should be" didn't sound good, if it's how it should be I'm ok with  that.